src/main/resources/reference.conf

```diff
@@ -17,5 +17,20 @@ resourceTypes =
           roleActions = ["view"]
         }
       ]
+    },
+    {
+      name = "billing-project"
+      actions = ["view", "alterpolicies", "readpolicies"]
```

Comment:
add 2 more actions launchbatchcompute and launchnotebook

an should we use underscores between words in actions?